lib/Sabberworm/CSS/CSSList/CSSList.php

```diff
@@ -97,7 +97,7 @@ private static function parseListItem(ParserState $oParserState, CSSList $oList)
 				}
 			}
 		} else {
-			return DeclarationBlock::parse($oParserState);
+			return DeclarationBlock::parse($oParserState, $oList);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Sabberworm\CSS\RuleSet\DeclarationBlock' in method 'parseListItem'.](https://app.codacy.com/app/sabberworm/PHP-CSS-Parser/pullRequest?prid=4484969)